src/data/faq_de.json

```diff
@@ -1887,6 +1887,67 @@
+                        ]
+                    },
+                    {
+                        "title": "Warum erhalte ich eine Risikowarnung für ein Datum, an dem ich keine Kontakte hatte / nicht das Haus verlassen habe?",
+                        "anchor": "risk_encounter_doubted",
+                        "active": false,
+                        "textblock": [
+                            "Risikobegegnungen können auch durch Nachbarn, Passanten und Personen, die sich vor Ihrem Haus aufhielten, durch Paketboten, beim Stehen im Stau neben anderen Autos an der Ampel oder auf der Autobahn, oder auch bei einem kurzen Einkauf entstehen. Die Bluetooth-Signale für die Abstandsmessung können hierbei durch Fenster, Hauswände oder Autotüren reichen. Die Risikokachel in der Corona-Warn-App spiegelt jedoch “nur” das Infektionsrisiko wider, nicht aber Ihren Infektionsstatus, denn die App kann natürlich <b>nicht</b> erkennen, ob und wie stark jemand infiziert oder infektiös ist.",
+                            "Siehe auch FAQ: <ul><li><a href='#risk_encounter_not_realtime' target='_blank'>Warum erhalte ich erst nach mehreren Tagen nach der Risikobegegnung eine Risikowarnung?</a></li><li><a href='#risk_encounter_no_warning' target='_blank'>Warum erhalte ich keine Risikowarnung, obwohl ich Kontakt zu einer Person hatte, die eine Risikobegegnung in ihrer App angezeigt bekommt?</a></li><li><a href='#risk_encounter_different_devices' target='_blank'>Wie kommt es zu Unterschieden in der Risikobewertung auf mehreren Geräten, auch wenn die jeweiligen Smartphones sich nicht weit voneinander entfernt befanden?</a></li><li><a href='#pos_test_no_risk_encounter' target='_blank'>Ich wurde positiv getestet. Warum erhält eine Person im gleichen Haushalt keine Risikowarnung?</a></li><li><a href='#risk_encounter_place' target='_blank'>Werden zu einer Risiko-Begegnung Personen und/oder Orte zugeordnet?</a></li></ul>"
+                        ]
+                    },
+                    {
+                        "title": "Werden zu einer Risiko-Begegnung Personen und/oder Orte zugeordnet?",
+                        "anchor": "risk_encounter_place",
+                        "active": false,
+                        "textblock": [
+                            "Im Kontakt-Tagebuch werden Ihnen unter anderem Begegnungen mit erhöhtem Risiko für den jeweiligen Tag aufgelistet. Durch das Tippen auf die Meldung der Begegnung gelangen Sie zu den in der App gespeicherten Personen und Orten (inklusive der Orte von gescannten Event-QR-Codes). Diese stehen in keinem Zusammenhang mit der von Ihnen erhaltenen Risikobegegnung.",
+                            "Durch Tippen auf einen Tag im Kontakt-Tagebuch gelangen Sie zu der Auswahl von Personen und Orten, welche Sie manuell in ihr Kontakt-Tagebuch eintragen können. Das Projektteam ist sich darüber bewusst, dass dies im Zusammenhang mit einer Risiko-Begegnung falsch aufgefasst werden kann und wird dies in Zukunft anpassen.",

```

Comment:
"manuell in ihr Kontakt-Tagebuch" =>
"manuell in Ihr Kontakt-Tagebuch"

grammar / capitalization